src/azure-cli-core/setup.py

```diff
@@ -55,7 +55,7 @@
     'argcomplete~=1.8',
     'azure-cli-telemetry',
     'colorama>=0.3.9',
-    'humanfriendly~=4.7',
+    'humanfriendly>=4.7,<9.0',
```

Comment:
@glaubitz, I have relaxed the dependency. Hope it will work on all Linux distros.